api/src/controllers/fields.ts

```diff
@@ -176,10 +193,14 @@ router.patch(
 			schema: req.schema,
 		});
 
-		const { error } = updateSchema.validate(req.body);
+		if (isSystemField(req.params['collection']!, req.params['field']!)) {
+			const { error } = systemUpdateSchema.validate(req.body, { abortEarly: false });
```

Comment:
Does it make sense here to return all validation errors (done with `abortEarly: false`) or just one error at the time?

Reply:
IMO yes, as otherwise they would need to do a one by one fix as we return which field is the issue instead of fixing all the relevant fields in one go.